services/watson-extension/src/watson_extension/clients/platform/chrome.py

```diff
@@ -0,0 +1,156 @@
+import abc
+import json
+import injector
+from dataclasses import dataclass
+from typing import List, Optional
+
+from common.identity import AbstractUserIdentityProvider
+from common.platform_request import AbstractPlatformRequest
+from watson_extension.clients import ChromeServiceURL
+
+
+@dataclass
+class Favorite:
+    id: str
+    pathname: str
+    favorite: bool
+    user_identity_id: str
+
+
+@dataclass
+class User:
+    account_id: str
+    first_login: bool
+    day_one: bool
+    last_login: str
+    last_visited_pages: List[dict]
+    favorite_pages: List[Favorite]
+    visited_bundles: dict
+
+
+@dataclass
+class Link:
+    id: Optional[str]
+    title: str
+    alt_title: Optional[List[str]]
+    links: Optional["Link"]
+    app_id: Optional[str] = None
+    is_group: Optional[bool] = None
+    is_external: Optional[bool] = None
```

Comment:
Are these really optional ? booleans can default to `False` unless we want to track if `is_group` is set or not.

Reply:
Defaulting to False sounds good. They are optional in the file, so I didn't think much about it :+1: 
![image](https://github.com/user-attachments/assets/a54ba5f9-750d-4f0c-ad6a-8babe1e28002)

Reply:
oh, nevermind then, the validation might fail if it is not present but required in our side